### PR TITLE
Removed all redundant links from suggested reading section of docs an…

### LINKED ISSUE #4314: remove redundant links from documentation
#### Current State
there are various links in total on meshery docs under **suggested reading** section which are redundant.
for example we are on **contributing to meshery server page** still a link to go on the same page itself is available under **sugested reading** section. We have few such cases on meshery docs.

#### Desired State
Remove such links from meshery docs under **sugested reading** section.
---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by shubham-tyagi**
removed all redundant links from contributing folder and from TOC of every doc spage.

suggested reading section of every page under project/contributing directory had a link to the page
itself which has been removed. Moreover, I also removed the redundant links in the TOC of every
page by commenting out the "include toc.html" line in page.html.

Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**

This PR fixes #4314

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e766f28263e363670d2ce8ff2b099364e2fcbc0b at: https://615ae428bb69fc4c2648d8e9--meshery-docs-preview.netlify.app

**Comment by shubham-tyagi** [comment]
## Observe the repetition of links in the TOC section of the page like to set up and run meshery on AKS, connect meshery to azure, etc. 
links before

## I have removed this repetition of links from every page as well apart from removing those links that were pointing to the page itself. 
links now

**Comment by shubham-tyagi** [comment]
#### Can you also add the hactoberfest-accept tag so that this pull request could be accepted in hacktoberfest if merged ?

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@shubham-tyagi good stuff, since you're doing this, could do them for all pages? you missed a few, like this one: https://615ae428bb69fc4c2648d8e9--meshery-docs-preview.netlify.app/guides/mesheryctl/authenticate-with-meshery-via-cli 

Also, if you're up for the challenge, you could try your hand at make a {% include %} for the suggested reading. Right now we have to write the `Suggested Reading` is all pages, it would be much easier if it was a reusable component. But, I'll leave that call to you, please proceed with whatever you feel comfortable with.

**Comment by shubham-tyagi** [comment]
@warunicorn19 yeah sure, I will do the same for all the pages and even after updating pull request if you see same scenario in some other page please feel free to point it out to me. I completely agree with you on making suggested reading a reusable component, this would implement the DRY methodology. So should I create another issue for the same ? because this issue is for removing redundant links.

**Comment by warunicorn19** [comment]
Yup @shubham-tyagi you're right, better to track in a different issue, I think there's an open issue for it already let me try to send you link of that.

**Comment by warunicorn19** [comment]
Okay I think that issue got closed😅, please create a new one for that and I'll assign it to you.

**Comment by shubham-tyagi** [comment]
yeah sure @warunicorn19, on it.

**Comment by l5io** [comment]
🚀 Preview for commit 9783fba409dc601ddfe0818b6fe4287593ae7c9d at: https://615b58d8a1e3f0370120a80f--meshery-docs-preview.netlify.app

**Comment by shubham-tyagi** [comment]
please suggest if there are some more changes that should be made in this PR @warunicorn19.

**Review by alphaX86 — CHANGES_REQUESTED** [review]
@shubham-tyagi nice work you're doing... but I'm reviewing your PR and noticed that one slight misalignment is made here...

image

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -142,38 +153,45 @@ Meshery documentation is made of these components:
...
+```
+$ make docker-docs
+```
 
 This doesn't require the need for installing Jekyll and Ruby in your system
 
-**But, you need to make sure that GNU make is working in your system (might not work in Windows)** 
+**But, you need to make sure that GNU make is working in your system (might not work in Windows)**
+
 #### Note
```
Comment (alphaX86) [inline docs/pages/project/contributing/contributing-docs.md:175]: Nice changes you made there for readability! GG!

Comment (shubham-tyagi) [reply docs/pages/project/contributing/contributing-docs.md:175]: Yeah that's my mistake, I will fix it. Thanks a lot for pointing it out @alphaX86

**Comment by l5io** [comment]
🚀 Preview for commit c92a2de2748d9ed48d0f731f663ac3d1c041fed8 at: https://615c0b6e7f5ec2c1f968b6b3--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md (hunk 1 above)
Comment (shubham-tyagi) [reply docs/pages/project/contributing/contributing-docs.md:175]: I have fixed the misalignment, please point out if there is any other issue with the PR @alphaX86

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉, Thanks @shubham-tyagi

**Comment by shubham-tyagi** [comment]
thanks to you both @warunicorn19 and @alphaX86 for helping and guiding me right through it.

**Review by alphaX86 — APPROVED** [review]
LGTM as well 👍

**Comment by shubham-tyagi** [comment]
I will keep that in mind going forward, thanks for the suggestion.  @alphaX86

**Comment by shubham-tyagi** [comment]
but can you go in this link https://docs.meshery.io/project/contributing-gitflow, in here it is written to rebase after merging in the master branch. Am i going in the wrong direction @alphaX86

**Comment by alphaX86** [comment]
> but can you go at this link https://docs.meshery.io/project/contributing-gitflow, in here it is written to rebase after merging in the master branch. Am i going in the wrong direction @alphaX86

Well, if you do rebase then you'd need to squash commits in order for clean git commit history

**Comment by shubham-tyagi** [comment]
okay, so we have two options either rebase and squash commits or directly merge in the branch we are working on. Is that correct? @alphaX86

**Comment by shubham-tyagi** [comment]
what do you suggest in this case, should I have a new pull request for this issue or leave it as it is ?

**Comment by alphaX86** [comment]
Better open a new PR as the commit history is huge and we are confused what file to look on now. And yes, rebase and squash commits

**Comment by shubham-tyagi** [comment]
on it @alphaX86.

**Pull request closed without merging**